traits_enaml/testing/tests/test_enaml_test_assistant.py

```diff
@@ -70,16 +70,21 @@ def test_find_all_enaml_widgets(self):
 
 class TestEnamlTestHelperFunctions(EnamlTestAssistant, unittest.TestCase):
 
+    def setUp(self):
+        super(TestEnamlTestHelperFunctions, self).setUp()
+        self.old_stdout = sys.stdout
+        self.stream = io.StringIO()
+        sys.stdout = self.stream
+
+    def tearDown(self):
+        sys.stdout = self.old_stdout
+        self.stream.close()
+        super(TestEnamlTestHelperFunctions, self).tearDown()
```

Comment:
I would do this with a context manager instead. I can never remember, does setUp/tearDown run for every test?

Reply:
Yes. the setUp/tearDown methods get run for every test method. Using a context manager would remove the duplication between the enaml test and the gui test. Ok. Rewriting.

Reply:
i've refactored the logic in setUp/tearDown methods into the context manager as discussed. can you take another look so i can merge?